src/Elasticsearch.Net/Purify/Purify.cs

```diff
@@ -120,33 +120,38 @@ static PurifierDotNet()
 			}
 
 			//Code inspired by Rasmus Faber's solution in this post: http://stackoverflow.com/questions/781205/getting-a-url-with-an-url-encoded-slash
-			public Uri Purify(Uri uri)
+			Uri IPurifier.Purify(Uri uri)

```

Comment:
I know this could be private here but we pretty much source import this from https://travis-ci.org/Purify-net/PUrify where this change makes less sense. 

Most of these refactorings make more sense as a PR to that repository :+1: